### PR TITLE
Add support for HEAD requests against navigated resources.

### PR DESCRIPTION
**Pull request by cank-coder**

### CODE REVIEW

File: src/halboy/navigator.clj [hunk 1]
```diff
@@ -181,6 +191,15 @@
   ([resource settings]
    (resource->Navigator resource settings)))
 
+(defn head
+  ([navigator link]
```
Comment (jimmythompson) [inline src/halboy/navigator.clj]: Can you add a comment to this method, so it is in line with other methods?

@ChloeAcheampong @tobyclemson

Comment (cank-coder) [reply src/halboy/navigator.clj]: They all have descriptions that aren't very HTTP oriented. We were going to go with, "Performs a HEAD request against a link in an API.". Does that work for you? Anything more in line with the others?

Comment (jimmythompson) [reply src/halboy/navigator.clj]: That would be great.

Comment (cank-coder) [reply src/halboy/navigator.clj]: Done.

**Comment by jimmythompson** [comment]
This is available as of 3.1.1.

**Comment by tobyclemson** [comment]
Thanks. You’re my hero.

**Pull request merged**